src/_partials/_navbar.erb

```diff
@@ -1,4 +1,7 @@
-<nav class="p-6 pt-8 lg:pt-0 lg:block" :class="{ 'hidden': !nav_open }">
+<%# in conditonal class of nav , 'w-sreen' can be also good but it will cover all of the screen and width of the navlink will get increases  %>
```

Comment:
??

Reply:
- sorry actually there was typo it is  'w-screen' class instead of 'w-sreen' which cover full width

```
<nav class="lg:p-6  pt-8 lg:pt-0 lg:w-auto lg:block fixed lg:static top-0 z-10 bg-white dark:bg-zinc-900 transition-all duration-500" 
     :class="{ 'w-[320px] p-6 ': nav_open, 'w-0 p-0 overflow-hidden': !nav_open }">
```
to
 ```
<nav class="lg:p-6  pt-8 lg:pt-0 lg:w-auto lg:block fixed lg:static top-0 z-10 bg-white dark:bg-zinc-900 transition-all duration-500" 
     :class="{ 'w-screen p-6 ': nav_open, 'w-0 p-0 overflow-hidden': !nav_open }">
```

- **issue**:  by adding 'w-screen' width it cover full width hence the width of the navigation links also get increases which may be not  good UX for width in range 500px to 1023px
- idea/solution request: we can add some max-width like 500px or more